### ISSUE TITLE
[Request] Scrape for English names of Torrents too and or embeds.

### ISSUE THREAD

**Issue by mathu204** [SwagOtaku/plugin.video.kaito#28]
Trying to play an episode of the first season for Fire Force with source select where I notice that the only links that come up are Embeds in the name of "Enen no Shouboutai". My Request is to add to Kaito the abililty to scrape in the English titles too bc there are many torrents under the name "Fire Force" and my cloud files are all under "Fire Force"

**Comment by SwagOtaku** [comment]
kaito already scrapes for english titles, it's just that the cloud inspection needed to be adjusted. I have an update with the adjustment ready to release, but as i'm writing this trakt is having issues. So i'm holding out releasing until trakt stabilizes since the new update will clear cached meta data and since trakt search api is down the addon won't be able to retrieve new metadata

**Comment by mathu204** [comment]
> kaito already scrapes for english titles, it's just that the cloud inspection needed to be adjusted. I have an update with the adjustment ready to release, but as i'm writing this trakt is having issues. So i'm holding out releasing until trakt stabilizes since the new update will clear cached meta data and since trakt search api is down the addon won't be able to retrieve new metadata

Oh that’s dope, here’s to a happy new year

**Comment by SwagOtaku** [comment]
released new update, let me know how it works

**Comment by mathu204** [comment]
> released new update, let me know how it works

Hey, thank you i will. Is there a changelog for other stuff that was changed/improved too? oh i might be wrong, but i think 0.0.7 seems to save episode progress now! Before if i restarted kodi it would be wiped or if i back out the addon and come back

**Comment by SwagOtaku** [comment]
are you using anilist, if yes i added episode progress for anilist but forgot to add it to changelog. You can view changelog here https://github.com/SwagOtaku/plugin.video.kaito/blob/main/changelog.txt or in kodi

**Comment by mathu204** [comment]
yes i am, so I tested on episode 1 of attack on titan. whne scraping it says 41 cached/41 torrents but only like 5-6 sources show up. The top 3 are incorrect while the bottom 2 are embeds from gogo. Here is a screenshot, and a comparison of sources for the same episode when using Wako/Helios for comparison. If you havent already i think AnimeTosho might be a good source to consider adding next.

![135752311_724133311572911_4392231301098216904_n](https://user-images.githubusercontent.com/52363082/103469049-eb363f00-4d2d-11eb-873a-7a34b1f8a35c.jpg)

![Screenshot 2021-01-02 190315](https://user-images.githubusercontent.com/52363082/103469042-cf329d80-4d2d-11eb-847d-6c25a639159c.jpg)

also, i might have to clear the cache and re test so let me do that too and come back

**Comment by mathu204** [comment]
Oh and also Im not sure if this may be of any help to you, I hope it does. There is a program called Taiga.moe for windows that is made for torrenting anime and stuff right, but it detects anime very very well im not sure if its possible but i think if u could take a look at its code it might help u alot with writing detection in Kaito as well. 
But anyways great to see u back on kaito at the start of the new year hope ur doing well and continue to stay well :D

**Comment by mathu204** [comment]
oh u knwo what i just realized this wasnt a fix for scraping just the cloud stuff lmao im an idiot, let me test the cloud content

**Comment by mathu204** [comment]
Yup my cloud sources get found now! thank you!!!

**Comment by lovecrafthowardphillips** [comment]
> are you using anilist, if yes i added episode progress for anilist but forgot to add it to changelog. You can view changelog here https://github.com/SwagOtaku/plugin.video.kaito/blob/main/changelog.txt or in kodi

Excellent new feature, thank you. Keep up the great work, easily the best anime addon, hands down.

**Comment by mathu204** [comment]
> are you using anilist, if yes i added episode progress for anilist but forgot to add it to changelog. You can view changelog here https://github.com/SwagOtaku/plugin.video.kaito/blob/main/changelog.txt or in kodi

Not sure what changed but my Fire Force is not being scanned by Kaito anymore from the library i am on 0.0.7
